src/common/template.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import fs from "fs-extra";
+import Handlebars from "handlebars";
+
+/**
+ * Holds a template and renders the template with the given data
+ */
+export class Template {
+  content: string;
+
+  /**
+   * Creates a Template object
+   * @param path - Path to the template file
+   * @param hb - Optional handlebars instance, default instance is used if not provided
+   */
+  constructor(public path: string, public hb = Handlebars) {
```

Comment:
```suggestion
  constructor(public path: string, public handlebars = Handlebars) {
```
Using the full name makes it more clear what the property actually is.

Reply:
I felt there are too many "handlebars" which is kind of confusing